src/chigame/games/views.py

```diff
@@ -0,0 +1,9 @@
+from django.views.generic import ListView
+
+from .models import Game
+
+
+class GameListView(ListView):
+    model = Game
+    queryset = Game.objects.all()
```

Comment:
Tiny and nit-picky comment but I think explicitly defining the `queryset` here is redundant because `ListView` automatically fetches all objects.